dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/HibernateTrackedEntityStore.java

```diff
@@ -556,6 +557,10 @@ private String joinAttributeValue(TrackedEntityQueryParams params) {
       String teav = "lower(" + col + ".value)";
       String ted = col + ".trackedentityid";
 
+      if (filters.getValue().stream().allMatch(qf -> qf.getOperator().isNull())) {
```

Comment:
I am confused by that `if`
You are in a loop, what if there are filters on 2 attributes and the first one has only `null` filter?
Would we skip the required `inner join` for the second attribute?

Reply:
Fixed, the result is the same, that's why no test failed.